stripe/src/main/java/com/stripe/android/PaymentSessionConfig.java

```diff
@@ -1,30 +1,30 @@
-package com.stripe.android.view;
+package com.stripe.android;
 
 import android.os.Parcel;
 import android.os.Parcelable;
 import android.support.annotation.NonNull;
 
 import com.stripe.android.model.Address;
 import com.stripe.android.model.ShippingInformation;
+import com.stripe.android.view.ShippingInfoWidget;
 
 import java.util.ArrayList;
 import java.util.List;
 
 /**
- * Class that tells {@link PaymentFlowActivity} what UI to render
+ * Class that tells {@link com.stripe.android.PaymentSession} what functionality it is supporting.
  */
-public class PaymentFlowConfig implements Parcelable {
-
+public class PaymentSessionConfig implements Parcelable {
 
     @NonNull private List<String> mHiddenShippingInfoFields;
     @NonNull private List<String> mOptionalShippingInfoFields;
     @NonNull private ShippingInformation mShippingInformation;
-    private boolean mHideShippingInfoScreen;
-    private boolean mHideShippingMethodsScreen;
+    private boolean mRequireShippingInfo;
```

Comment:
nit: let's change the name to `mShippingInfoRequired` and `mShippingMethodsRequired`, mainly so that the getters make more grammatical sense (`isShippingInfoRequired` versus `isRequireShippingInfo`)

Reply:
ah yes, that is much nicer!